CONTRIBUTING.md

```diff
@@ -1,123 +0,0 @@
-# Contributing to Eclipse Theia
```

Comment:
@planger We don't have something similar yet for CDT.cloud so I deleted this.

Reply:
Right, we should add a dedicated `CONTRIBUTING.md` later.